spec/spec_helper.rb

```diff
@@ -1 +1,9 @@
 require 'puppetlabs_spec_helper/module_spec_helper'
+require 'rspec-puppet'
```

Comment:
This is implicitly included by the spec_helper above, probably also the whole bloc below (not sure about that)